evms/besu.go

```diff
@@ -112,7 +112,13 @@ func (vm *BesuVM) ParseStateRoot(data []byte) (string, error) {
 		root := string(data[start : start+2+64])
 		return root, nil
 	}
-	return "", errors.New("besu: no stateroot found")
+	start = strings.Index(string(data), `"stateRoot":"`)
```

Comment:
Do you have any example statetest which makes besu emit this? Or is that an opcoming change?

Reply:
https://github.com/hyperledger/besu/pull/8006 - will land in main after new year